Project/Library/LibraryViewController.swift

```diff
@@ -8,22 +8,63 @@
 
 import Foundation
 import UIKit
-class LibraryViewController: UIViewController {
+class LibraryViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    private let _viewModel : BookViewModel = BookViewModel()
+    private let _view: BooksTableView = BooksTableView.loadFromNib()!
+    private var books : [Book] = []
```

Comment:
This attribute isn't needed, as you can call the 'getter' method locally when you need it. If you do it this way, you lose transparency (picture twenty attributes with different names, all using data from the view model, for example).

It's not wrong to use something similar in some method, like `var books = _viewModel.getBooks()`and then just use it there.